desktop/windows/release-notes/index.md

```diff
@@ -34,28 +34,27 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Security
 
-- Fixed [CVE-2021-45449](https://docs.docker.com/security/#cve-2021-45449) that affects users currently on Docker Desktop version 4.3.0 or 4.3.1.
+- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774) where arbitrary files could be moved on the system.
 
-Docker Desktop version 4.3.0 and 4.3.1 has a bug that may log sensitive information (access token or password) on the user's machine during login.
-This only affects users if they are on Docker Desktop 4.3.0, 4.3.1 and the user has logged in while on 4.3.0, 4.3.1. Gaining access to this data would require having access to the user’s local files.
+### Bug fixes and minor changes
 
-### New
+- Fixed logging in from WSL 2. Fixes [docker/for-win#12500](https://github.com/docker/for-win/issues/12500).
 
-- Easy, Secure sign in with Auth0 and Single Sign-on
-  - Single Sign-on: Users with a Docker Business subscription can now configure SSO to authenticate using their identity providers (IdPs) to access Docker. For more information, see [Single Sign-on](../../../single-sign-on/index.md).
-  - Signing in to Docker Desktop now takes you through the browser so that you get all the benefits of auto-filling from password managers.
+### Known issues
 
-### Upgrades
+- Clicking «Proceed to Desktop» after logging in in the browser, sometimes does not bring the Dashboard to the front.
```

Comment:
extra "in"

Reply:
Good catch. PR on its way ;)

Reply:
Isn’t that «logging in» then «in the browser»? 🤔

Reply:
I suppose it should actually be «signing in» as I think we’re trying to use that consistently now…

Reply:
It's logging in "in a browser".. Updating it to "logging in through the browser" to avoid repeating the word 'in'.

Reply:
PR here - https://github.com/docker/docker.github.io/pull/14170